main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/gruntwork-io/aws-nuke/internal/awsnuke"
+	"github.com/gruntwork-io/gruntwork-cli/entrypoint"
+	"github.com/gruntwork-io/gruntwork-cli/logging"
+	"github.com/gruntwork-io/gruntwork-cli/shell"
+	"github.com/urfave/cli"
+)
+
+var VERSION string
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "aws-nuke"
+	app.Author = "Gruntwork <www.gruntwork.io>"
+	app.Version = VERSION
+
+	app.Flags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "non-interactive",
+			Usage: "Assume yes for all interactive prompts",
+		},
+	}
+
+	log := logging.GetLogger("aws-nuke")
+	app.Action = func(c *cli.Context) error {
+		if c.NArg() < 1 {
+			return cli.NewExitError("insufficient number of arguments", 1)
+		}
+
+		id := c.Args().Get(0)
+
+		nuke := awsnuke.New(id)
```

Comment:
I'd split this into several named functions to make it easier to read (and debug and test): e.g., `showInstances`, `confirmDelete`, `deleteInstances`.